src/pages/guides/using/idcs/idcs-events-data-stream-setup.md

```diff
@@ -0,0 +1,258 @@
+            "eventId": "f1dbc354-3878-4273-8a9e-c2a6b57543cc",
+            "state": "ASSETS_DOWNLOAD_STARTED",
+            "timestamp": 1720631448826
+          }
+        },
+        "id": "70b8efcb-38f2-4781-84ee-036013d3f5bc",
+        "source": "urn:uuid:5c8f2d48-a2a7-4d0e-bb49-aabf9a72bd1e",
+        "specversion": "1.0",
+        "type": "ASSETS_DOWNLOAD_STARTED",
+        "datacontenttype": "application/json",
+        "time": "2024-07-10T17:10:49.165Z",
+        "event_id": "d1c47805-a971-4685-8e63-863ba4ecd325",
+        "recipient_client_id": "7af7ab96045640b5a6e4724810145eb9"
+      }
+    },
+    {
+      "position": "tuna:a1db5e65-7863-4110-bd45-2c4d2076871b.salmon:e1a7e7e0-39e2-453f-8c72-d70725020941.1bf6e136-967f-47c3-a64e-afe90f6656d0.1.1720631450.krwtyp81wqfhdbmbhpqa",
+      "event": {
+        "data": {
+          "key": "2196c45f-8721-4587-8136-b92147a5efa6",
+          "source": "indesign",
+          "value": {
+            "jobId": "8b524c70-6779-47f1-ac2a-ce93c23b3f5f",
+            "eventId": "2196c45f-8721-4587-8136-b92147a5efa6",
+            "state": "ASSET_DOWNLOAD_STARTED",
+            "timestamp": 1720631448998,
+            "data": {
+              "destination": "b4adb91c-806a-4b80-92b9-e958a6bb1f13",
+              "source": {
+                "type": "CAPABILITY_BUNDLE",
+                "url": "stage/capability-bundles/indesign/rendition/pdf/1.0.46/67a46077-7944-4100-9236-35bb739baa9b"
+              }
+            }
+          }
+        },
+        "id": "740798e9-0827-4efd-a769-78b9a67c0ee8",
+        "source": "urn:uuid:5c8f2d48-a2a7-4d0e-bb49-aabf9a72bd1e",
+        "specversion": "1.0",
+        "type": "ASSET_DOWNLOAD_STARTED",
+        "datacontenttype": "application/json",
+        "time": "2024-07-10T17:10:49.238Z",
+        "event_id": "1bf6e136-967f-47c3-a64e-afe90f6656d0",
+        "recipient_client_id": "7af7ab96045640b5a6e4724810145eb9"
+      }
+    }
+  ],
+  "_page": {
+    "last": "tuna:a1db5e65-7863-4110-bd45-2c4d2076871b.salmon:e1a7e7e0-39e2-453f-8c72-d70725020941.1bf6e136-967f-47c3-a64e-afe90f6656d0.1.1720631450.krwtyp81wqfhdbmbhpqa",
+    "count": 3
+  }
+}
+````
+
+*Example Event (single)*
+
+````json
+{
+  "events": [
+    {
+      "position": "tuna:e54a7fa0-eb13-4ad5-964c-5d682f03a188.salmon:3279e19f-7150-4e07-a223-795fb6c6fe8e.f60a502e-c16b-49c6-b3c1-770666a1921f.0.1720631451.o5nlq_28cmu1ajdkuhxc",
+      "event": {
+        "data": {
+          "key": "70db1be2-cdde-4a00-bc41-5acd33db3cc7",
+          "source": "indesign",
+          "value": {
+            "jobId": "8b524c70-6779-47f1-ac2a-ce93c23b3f5f",
+            "eventId": "70db1be2-cdde-4a00-bc41-5acd33db3cc7",
+            "state": "ASSET_DOWNLOAD_FAILED",
+            "timestamp": 1720631449367,
+            "data": {
+              "destination": "doc.indd",
+              "source": {
+                "assetId": "urn:aaid:sc:VA6C2:6b14edb7-7acd-34ad-825e-b70c3b2569f4",
```

Comment:
Since ACP is not exposed for external clients, we should not use assetId in the sample events.
We can update it to "url" format

Reply:
Lets remove assetid from all places and use source: { "url" } format

Reply:
Done